saos-common/src/test/java/pl/edu/icm/saos/common/json/JsonUtilsTest.java

```diff
@@ -30,24 +39,30 @@ public void before() throws JsonParseException, IOException {
     
     
     @Test
-    public void nextNode_SingleElement() throws JsonParseException, IOException {
+    public void nextObject_ClassObject() throws JsonParseException, IOException {
         
         // Given
-        String content = normalizeJson("{bre: 'sss', arr:['1112', 'abc']}");
+        String content = normalizeJson("{name: 'sss', somelist:['1112', 'abc']}");
         
         JsonParser jsonParser = jsonFactory.createParser(content);
         
         
         // Execute
-        String nodeStr = JsonUtils.nextNode(jsonParser);
+        JsonItem jsonItem = jsonUtils.nextJsonObject(jsonParser, JsonItem.class);
 
         // Assert
-        assertEquals(content.replaceAll("\\s", ""), nodeStr);
+        assertNotNull(jsonItem);
+        assertEquals("sss", jsonItem.getName());
+        assertThat(jsonItem.getSomelist(), Matchers.contains("1112", "abc"));
+        
+        
+        // Execute 2
+        assertNull(jsonUtils.nextJsonObject(jsonParser, JsonItem.class));
```

Comment:
I'll leave it as it is... I want to check it the next invocation will return null (i think it's not a wrong place here to do it).